### ISSUE TITLE
Scaladoc issues on Schedule

### ISSUE THREAD

**Issue by iravid** [zio/zio#1166]
```
[error] /home/circleci/project/core/shared/src/main/scala/zio/Schedule.scala:80:3: The link target "ZSchedule.doUntil" is ambiguous. Several members fit the target:
[error] [A, B](pf: PartialFunction[A,B]): zio.Schedule[A,Option[B]] in object ZSchedule [chosen]
[error] [A](f: A => Boolean): zio.Schedule[A,A] in object ZSchedule
[error] 
[error] 
[error] Quick crash course on using Scaladoc links
[error] ==========================================
[error] Disambiguating terms and types: Prefix terms with '$' and types with '!' in case both names are in use:
[error]  - [[scala.collection.immutable.List!.apply class List's apply method]] and
[error]  - [[scala.collection.immutable.List$.apply object List's apply method]]
[error] Disambiguating overloaded members: If a term is overloaded, you can indicate the first part of its signature followed by *:
[error]  - [[[scala.collection.immutable.List$.fill[A](Int)(⇒A):List[A]* Fill with a single parameter]]]
[error]  - [[[scala.collection.immutable.List$.fill[A](Int,Int)(⇒A):List[List[A]]* Fill with a two parameters]]]
[error] Notes:
[error]  - you can use any number of matching square brackets to avoid interference with the signature
[error]  - you can use \\. to escape dots in prefixes (don't forget to use * at the end to match the signature!)
[error]  - you can use \\# to escape hashes, otherwise they will be considered as delimiters, like dots.
[error]   /**
[error]   ^
[error] /home/circleci/project/core/shared/src/main/scala/zio/Schedule.scala:74:3: The link target "ZSchedule.doUntil" is ambiguous. Several members fit the target:
[error] [A, B](pf: PartialFunction[A,B]): zio.Schedule[A,Option[B]] in object ZSchedule [chosen]
[error] [A](f: A => Boolean): zio.Schedule[A,A] in object ZSchedule
[error] 
[error]   /**
[error]   ^
[error] two errors found
```

**Comment by adamgfraser** [comment]
I’m happy to take this as my first issue.

**Comment by iravid** [comment]
Please do, thanks @adamgfraser

**Comment by iravid** [comment]
Oh sorry @adamgfraser , looks like Kai has already solved this because it was blocking the release.

**Comment by iravid** [comment]
Resolved in #1171
